kodein-android/src/main/kotlin/com/github/salomonbrys/kodein/android/AndroidScopes.kt

```diff
@@ -2,11 +2,38 @@ package com.github.salomonbrys.kodein.android
 
 import android.app.Activity
 import android.app.Application
+import android.app.Fragment
+import android.app.Service
+import android.content.BroadcastReceiver
+import android.content.Context
 import android.os.Bundle
 import com.github.salomonbrys.kodein.*
 import com.github.salomonbrys.kodein.android.activityScope.lifecycleManager
 import java.util.*
+import android.support.v4.app.Fragment as SupportFragment
 
+/**
+ * Map that associates a ScopeRegistry to a context.
+ *
+ * Because it's a weak hash map, this prevents the context from being leaked.
```

Comment:
For `Context` I put leaked instead of destroyed because there is no public API for a `Context` getting "destroyed" as opposed to `Activity` or `Service` where there is an `onDestroy` method. I can change it to destroyed though.